subworkflows/local/utils_nfcore_zippypretext_pipeline/main.nf

```diff
@@ -64,36 +64,44 @@ workflow PIPELINE_INITIALISATION {
     )
 
     //
-    // Custom validation for pipeline parameters
+    // Create channel from fasta file provided through params.fasta
     //
-    validateInputParameters()
 
-    //
-    // Create channel from input file provided through params.input
-    //
+    input     = Channel.fromPath(
+                params.fasta,
```

Comment:
You should pass these params into the workflow directly in the `take:` section and then refer to them.

```suggestion
                fasta,
```

Same for agp, idxfile and hicmap below. This will be good future-proofing for upcoming versions of nextflow where params are only available in top-level workflows